nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_recovery.xml

```diff
@@ -15,8 +15,7 @@
         <FollowPath path="${path}"/>
       </Sequence>
       <SequenceStar name="RecoveryActions">
-        <clearEntirelyCostmapServiceRequest service_name="/local_costmap/clear_entirely_local_costmap"/>
-        <clearEntirelyCostmapServiceRequest service_name="/global_costmap/clear_entirely_global_costmap"/>
+        <ClearCostmaps/>
```

Comment:
This removes the ability to clear costmaps individually. Is that what we want?

Reply:
There's now a parameter to set which costmaps to clear from a call. More specifically it sets a vector of N costmaps to clear when called in case you have >2 costmaps.

Reply:
OK, frankly that was also something I hadn't thought about. 

I think another potential option is to have the clear costmap recovery the way it currently exists, but the BT clear costmaps action has the option to send a single costmap, or do all. As you suggest @mhpanah so it could be called as `<ClearCostmaps param="${local_costmap}"/>` or `<ClearCostmaps/>` where lack of param is a call to just do all in the parameter of the costmap recovery which has the parameter for all available ones. We couldn't just take the string service call from the BT `_action.h` because then we'd be building service clients on the fly and given how discovery is...

I don't like having a recovery embedded inside the BT, that's the real goal - to remove clear costmaps from the BT and move it with the rest of the recoveries.